config/300-service.yaml

```diff
@@ -27,4 +27,7 @@ spec:
     - all
     - knative
     - serving
+    shortNames:
+    - kservice
+    - ksvc
```

Comment:
I believe @n3wscott tried this and these didn't work?

Reply:
It works now!!! (using head version of kubectl)